util/settings/settings_test.go

```diff
@@ -233,6 +233,141 @@ func TestSettingsManager_GetResourceOverrides_with_empty_string(t *testing.T) {
 	assert.Len(t, overrides, 1)
 }
 
+func TestGetResourceOverrides_with_split_keys(t *testing.T) {
```

Comment:
I think test names are a bit confusing. I would rename to them as following:

`TestGetResourceOverrides_with_split_keys` 
to
`TestGetResourceOverrides`

`t.Run("TestGetResourceOverrides"`  
to
`t.Run("MergedKey"`

`t.Run("TestGetResourceOverrides_with_split_keys"` 
to
`t.Run("SplitKeys"`

`t.Run("TestGetResourceOverrides_with_split_keys_compareoptions_all"`
to
`t.Run("SplitKeysCompareOptionsAll"`

Reply:
I cannot rename the following as we already have another test with the name `TestGetResourceOverrides`

`TestGetResourceOverrides_with_split_keys`
to
`TestGetResourceOverrides`

https://github.com/argoproj/argo-cd/pull/6060/files#diff-450b74ca5090fe496ffae80e53533e2786df3fb9942eb99d46decd6e01e941bcL144

Reply:
Got it. Updated test names look good. Thank you!